applications/nrf_desktop/README.rst

```diff
@@ -1132,7 +1132,8 @@ nRF52840 Gaming Mouse (nrf52840gmouse_nrf52840)
         * Bluetooth is configured to use Nordic's SoftDevice link layer.
 
       * The configuration with the B0 bootloader is set as default.
-      * The board supports debug :ref:`nrf_desktop_bluetooth_guide_fast_pair` configuration (:file:`prj_fast_pair.conf`).
+      * The board supports debug (:file:`prj_fast_pair.conf`) and release (:file:`prj_release_fast_pair.conf`) :ref:`nrf_desktop_bluetooth_guide_fast_pair` configurations.
+        Both configurations use MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and support firmware updates using both :ref:`nrf_desktop_dfu` and :ref:`nrf_desktop_dfu_mcumgr`.
```

Comment:
```suggestion
        Both configurations use the MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``), and support firmware updates using the :ref:`nrf_desktop_dfu` and :ref:`nrf_desktop_dfu_mcumgr` modules.
```
We could use the shortcut option here, which could be added to the shortcut.txt file.

Reply:
Also I think we should not duplicate `modules` word here.

Reply:
The reason to add "modules" (missed to see other modules have "module" as suffix :) ) was because "nrf_desktop_dfu_mcumgr" didn't have the suffix "module". 
I agree with the comment, and we could add a module suffix to "nrf_desktop_dfu_mcumgr" like we have for the other modules.

Reply:
> Is it worth to introduce shortcut for the documentation sentence that appears twice? The gaming mouse has is slightly modified.

Agree with you. We can leave this as is.